### PR TITLE
Randomizer Module

### PR DESCRIPTION
**Pull request by PSNAppz**
### Changes Proposed 
- Tor identity Randomizer 

### Explanation of Changes
Randomizes tor identity for every request or every 'n' requests.

### CODE REVIEW

File: modules/randomizer.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+"""
+Module for randomizing tor identity
+"""
+import socket
+
+def randomize_IP(HOST, PORT, PWD):
+
+	try:
+    	tor_c = socket.create_connection((HOST, PORT))
```
Comment (KingAkeem) [inline modules/randomizer.py]: This line to 13 should be indented.

File: modules/randomizer.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    	tor_c.send('AUTHENTICATE "{}"\r\nSIGNAL NEWNYM\r\n'.format(PWD))
+    	response = tor_c.recv(1024)
+    	if response != '250 OK\r\n250 OK\r\n':
+        	sys.stderr.write('Unexpected response from Tor control port: {}\n'.format(response))
+	except Exception, e:
+    	sys.stderr.write('Error connecting to Tor control port: {}\n'.format(repr(e)))
```
Comment (KingAkeem) [inline modules/randomizer.py]: Should be indented.

File: modules/randomizer.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+
+
+#def randomize_header():
```
Comment (KingAkeem) [inline modules/randomizer.py]: What's this for?

**Comment by PSNAppz** [comment]
@KingAkeem This commit was a pure copy paste. I haven't tested it yet. New changes are yet to be pushed.

**Comment by atlking-Novetta** [comment]
@PSNAppz Ok, I will review this whenever it's finally ready.

**Comment by abhi1998das** [comment]
aa

**Comment by KingAkeem** [comment]
Closed until ready to be worked on.

**Pull request closed without merging**